package.json

```diff
@@ -60,7 +60,7 @@
     "stylelint-config-recommended": "5.0.0",
     "stylelint-config-styled-components": "0.1.1",
     "stylelint-processor-styled-components": "1.10.0",
-    "typescript": "4.6.3"
+    "typescript": "4.8.2"
```

Comment:
Apart from upgrading to 4.7+ 
If consumer is using web target then no change is needed. Since by default it will be picked. 
If they use react-native then they need to add that `moduleSuffix` option. I'll add docs for this.

Reply:
Yes we should.